keps/sig-storage/1432-volume-health-monitor/README.md

```diff
@@ -674,8 +679,180 @@ _This section must be completed when targeting alpha to a release._
   disable this feature is to install or unistall the sidecars, we cannot write
   tests for feature enablement/disablement.
 
+### Rollout, Upgrade and Rollback Planning
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can a rollout fail? Can it impact already running workloads?**
+  Try to be as paranoid as possible - e.g., what if some components will restart
+   mid-rollout?
+   This feature does not have a feature gate. It is enabled when the health
+   monitoring controller and agent sidecars are deployed with the CSI driver.
+   So the only way for a rollout to fail is that deploying the health
+   monitoring controller or agent sidecars with the CSI driver fails. If
+   the health monitoring controller cannot be deployed, no events on volume
+   condition will be reported on PVCs. If the health monitoring agent cannot
+   be deployed, no event on volume condition will be reported on the pod.
+
+* **What specific metrics should inform a rollback?**
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  Describe manual testing that was done and the outcomes.
+  Longer term, we may want to require automated upgrade/rollback tests, but we
+  are missing a bunch of machinery and tooling and can't do that now.
+  Manual testing will be done.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs,
+fields of API types, flags, etc.?**
+  Even if applying deprecation policies, they may still surprise some users.
+  No.
+
+### Monitoring Requirements
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can an operator determine if the feature is in use by workloads?**
+  Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+  checking if there are objects with field X set) may be a last resort. Avoid
+  logs or events for this purpose.
+  An operator can check the metrics `csi_sidecar_operations_seconds`,
```

Comment:
is this divided by the type of operation?  This KEP appears to be adding APIs and knowing which APIs are being used seems to have value.

Reply:
This is per sidecar.  For the health monitoring controller sidecar, `csi_sidecar_operations_seconds` will be measuring ListVolumes or GetVolume RPC. For the health monitoring agent sidecar, `csi_sidecar_operations_seconds` will be measuring NodeGetVolumeStats RPC.

Reply:
> This is per sidecar. For the health monitoring controller sidecar, `csi_sidecar_operations_seconds` will be measuring ListVolumes or GetVolume RPC. For the health monitoring agent sidecar, `csi_sidecar_operations_seconds` will be measuring NodeGetVolumeStats RPC.

Can you indicate that its necessary to slice the metric by process after you aggregate?  It would have helped me.